src/core.js

```diff
@@ -46,7 +46,15 @@ let Game = {
    *     },
    *     guesses: 1                 // Not present if no prompt set
    *   },
-   *   winner : 'red|blue'          // Not present if neither team has won yet
+   *   winner: 'red|blue',         // Not present if neither team has won yet
```

Comment:
random idea: if you do this in typescript, you get compile time checking, and also don't have to worry about this going out of date